app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from typing import List
+
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: List[Player]) -> int:
+    """Calculate and return total team rating."""
+    return sum([p.get_rating() for p in players])
```

Comment:
Code Efficiency: Avoid creating a redundant list by passing a comprehension without square brackets to the sum function. Instead, use a generator expression directly.